lib/ansible/plugins/action/__init__.py

```diff
@@ -175,19 +175,32 @@ def _configure_module(self, module_name, module_args, task_vars=None):
         final_environment = dict()
         self._compute_environment_string(final_environment)
 
+        become_kwargs = {
+            'become': False,
+            'become_method': None,
+            'become_user': None,
+            'become_password': None,
+            'become_flags': None
```

Comment:
Instead of setting these all to their defaults from `modify_module`, just use `become_kwargs = {}` to start, it will achieve the same goal without duplicating defaults that could lead to issues later.